client/src/PlayerLanding/index.tsx

```diff
@@ -34,15 +35,14 @@ function PlayerLanding() {
       return;
     }
     // They have joined, but have been removed from the server
-    if (!role) {
```

Comment:
ew jesus this is bad.  Don't use effects.  they are nothing but trouble

Reply:
But but you wrote that

Reply:
But you're right I should have removed it soon as I saw it.

Reply:
What? I wrote the effect???

Reply:
Oh this is for kicked, sorry.  am dumb.  But now that we have hasJoined it's not relevant

Reply:
Wait, when is hasJoined ever different than than `useMe()`?